scripts/index.js

```diff
@@ -0,0 +1,126 @@
+const state={
+    color:"teal",
+    temp:71,
+    garden:"☘️🌻🌵💐",
+    
+};
+
+const increatTemp = ()=>{
+    const tempContainer=document.getElementById("temp")
+    state.temp +=1;
+    tempContainer.textContent=`${state.temp}`;
+    
+    updateColorofTemp(state.temp);
+    updateGarden(state.temp);
+};
```

Comment:
I just want to note that you could combine `increatTemp` and `decreatTemp` by adding a parameter

```suggestion
const changeTemp = (increment)=>{
    const tempContainer=document.getElementById("temp")
    state.temp +=increment;
    tempContainer.textContent=`${state.temp}`;
    
    updateColorofTemp(state.temp);
    updateGarden(state.temp);
};
```

Then you can register the event handler with an anonymous function like:

```javascript
increaseTempValue.addEventListener("click", () => changeTemp( +1 ) );
decreaseTempValue.addEventListener("click", () => changeTemp( -1 ) );

```